src/evergreen/evg_task_history.rs

```diff
@@ -286,6 +288,21 @@ fn hook_hook_name(identifier: &str) -> &str {
     identifier.split(HOOK_DELIMITER).last().unwrap()
 }
 
+/// Normalize the given test files.
+///
+/// Converts windows path separators (\) to unix style (/).
+///
+/// # Arguments
+///
+/// * `test_file` - test file to normalize.
+///
+/// # Returns
+///
+/// Normalized test file.
+fn normalize_test_file(test_file: &str) -> String {
+    test_file.replace('\\', "/")
```

Comment:
This is fine, but just to ask - is there anything built into Rust to convert paths between OS styles automatically like the Python Path library?

Reply:
Yes, rust has [Path](https://doc.rust-lang.org/stable/std/path/struct.Path.html) and `PathBuf`s for dealing with paths. We might be able to use them here, but we need strings because we are using them as identifiers to track runtimes, not as actual paths. So converting them back and forth would be annoying. I know we do similar things with python code in some places.